accelerator/core/response/ta_fetch_buffered_request_status.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef RESPONSE_TA_fetch_buffered_request_status_H_
+#define RESPONSE_TA_fetch_buffered_request_status_H_
+
+#include "common/macros.h"
+#include "common/model/bundle.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file accelerator/core/response/ta_fetch_buffered_request_status.h
+ */
+
+/**
+ * The enumeration of the status of buffered requests
+ */
+typedef enum {
+  UNSENT /**< Buffered request hasn't be broadcasted */,
```

Comment:
Add `MAM_` prefix for each entry. That is, `MAM_UNSET`, `MAM_NOT_EXIST` and `MAM_SENT`.

Reply:
The enum of request_with_uuid_status_t also been used in the api_fetch_buffered_request_status. Should we change it?

Reply:
If we add the `MAM_` prefix, the enum of `request_with_uuid_status_t` inside `ta_fetch_txn_with_uuid` may be misleading.

Reply:
How about `MAM_BUFREQ_`?